Makefile

```diff
@@ -19,6 +19,21 @@ build: deps
 		&& export CGO_ENABLED=0 \
 		&& go build -v -mod=vendor -ldflags $(BUILD_FLAGS) -o ${BINARY} ./cli/main.go
 
+install: build
+	@echo "=> Preparing directories and configs"
+	@id neo-go || useradd -s /usr/sbin/nologin -d /var/lib/neo-go neo-go \
+		&& mkdir -p /var/lib/neo-go \
+		&& chown -R neo-go:neo-go /var/lib/neo-go
+	@echo "=> Installing systemd service"
+	@mkdir -p /etc/neo-go \
```

Comment:
Please use `$(DESTDIR)` to be able to override paths, like
```
mkdir -p $(DESTDIR)/etc/neo-go
```
Or even better:
```
mkdir -p $(DESTDIR)$(sysconfdir)/neo-go
```
Same thing for `$(bindir)` and `$(systemd_unitdir)`. You'll need to add defaults to the `Makefile`, but in the end all of these things will be overridable and every package maintainer will just love you for that.

Reply:
done